Formula/deno.rb

```diff
@@ -0,0 +1,35 @@
+class Deno < Formula
+  desc      "Secure TypeScript runtime on V8"
+  homepage  "https://github.com/denoland/deno/"
+  url       "https://github.com/denoland/deno.git",
+      :tag      => "v0.2.5",
+      :revision => "320577c131c8058ba0065f26119ce1ce18bc23d2"
+
+  bottle do
```

Comment:
This bottle section should be remove in the new formula.

Reply:
As mentioned [above](https://github.com/Homebrew/homebrew-core/pull/35645#issuecomment-451073076) this bottle section is required to pass the tests. See error message in that comment.